src/main/kotlin/com/exactpro/th2/pico/operator/generator/impl/MqConfigHandler.kt

```diff
@@ -40,7 +40,8 @@ class MqConfigHandler(private val resource: BoxResource) : ConfigHandler() {
             ResourceType.Th2Estore to MqRouterConfigFactoryEstore(schemaName),
             ResourceType.Th2Mstore to MqRouterConfigFactoryMstore(schemaName),
             ResourceType.Th2Box to MqRouterConfigFactoryBox(schemaName),
-            ResourceType.Th2CoreBox to MqRouterConfigFactoryBox(schemaName)
+            ResourceType.Th2CoreBox to MqRouterConfigFactoryBox(schemaName),
+            ResourceType.Th2Job to MqRouterConfigFactoryBox(schemaName)
```

Comment:
How could we run jobs in pico?

Reply:
This part of pico is just a config generation. actually running the job would be the responsibility of pico runner app which is a separate project. The config generator will just generate all needed configs similar to the regular th2 box.